### PR TITLE
Update API docs index.md links + extra constants

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

File: docs/constants.md [hunk 1]
```diff
@@ -52,12 +52,15 @@ title: Constants
...
 * `users` - Logged-in users
+* `groups` - Groups
 * `guests` - Guest without a login
 * `emails` - A guest invited by email address
 
 ### Actor types of chat messages
 * `guests` - guest users
 * `users` - logged-in users
+* `emails` - users invited by email
+* `groups` - groups
```
Comment (nickvergessen) [inline docs/constants.md:63]: actually groups can't be author of chat messages.

Comment (PVince81) [reply docs/constants.md:63]: was this a mistake ? https://github.com/nextcloud/spreed/blob/master/src/constants.js#L54

Comment (nickvergessen) [reply docs/constants.md:63]: Attendees can have such actor types, but not chat messages.

Comment (PVince81) [reply docs/constants.md:63]: ah, got it. I never realized that actor type constants were used in two places, I'll remove that line

Comment (PVince81) [reply docs/constants.md:63]: https://github.com/nextcloud/spreed/pull/5722

**Pull request merged**